### PR TITLE
Use client Count API rather than obtain all records and counting client-side.

### PR DESCRIPTION
**Pull request by richard3d**
Small PR per observations from @larskendall, title says it all.

### CODE REVIEW

**Review by AnthonyWhitaker — APPROVED** [review]

File: packages/client/src/components/pages/home-page/home-page.tsx [hunk 1]
```diff
@@ -138,20 +135,10 @@ export const HomePage = () => {
...
-            limit: '',
-            unit: undefined,
-          },
-        }));
-      } catch (error) {
-        void dispatch(Modal.alert('Get Orphaned Records', formatErrorMessage(error, 'Failed to get orphaned records')));
-      }
+      }    
+      const orcResponse = await OrphanedRecordClient.getOrphanedRecordsCount(orgId);
+      setOrphanedRecordCount(orcResponse.count);
```
Comment (larskendall) [inline packages/client/src/components/pages/home-page/home-page.tsx:140]: This data is actually already in the redux store, and can be refreshed via an action. So you shouldn't need to store it as state in the home page. Here's an example of what that would look like...

```typescript
void dispatch(OrphanedRecordActions.fetchCount({ orgId: orgId! }));
```

File: packages/client/src/components/pages/home-page/home-page.tsx [hunk 2]
```diff
@@ -121,12 +117,13 @@ export const HomePage = () => {
...
-  const orphanedRecords = useAppSelector(OrphanedRecordSelector.root);
-  const [accessRequests, setAccessRequests] = useState<ApiAccessRequest[]>([]);
-  const [musterComplianceLastTwoWeeks, setMusterComplianceLastTwoWeek] = useState<number[]>([1.0, 0.0]);
   const favoriteDashboards = useAppSelector(UserSelector.favoriteDashboards)!;
   const workspaces = useAppSelector(UserSelector.workspaces)!;
   const dashboards = useAppSelector(WorkspaceSelector.dashboards)!;
+  
+  const [accessRequests, setAccessRequests] = useState<ApiAccessRequest[]>([]);
+  const [musterComplianceLastTwoWeeks, setMusterComplianceLastTwoWeek] = useState<number[]>([1.0, 0.0]);
+  const [orphanedRecordCount, setOrphanedRecordCount] = useState<number>(0);
```
Comment (larskendall) [inline packages/client/src/components/pages/home-page/home-page.tsx:126]: These types will be automatically inferred automatically. Just a heads up.

**Review by larskendall — CHANGES_REQUESTED** [review]

**Comment by richard3d** [comment]
@larskendall can you confirm this is good now?

File: packages/client/src/components/pages/home-page/home-page.tsx [hunk 3]
```diff
@@ -140,6 +141,7 @@ export const HomePage = () => {
       }
 
       try {
+        await dispatch(OrphanedRecordActions.fetchCount({ orgId: orgId! }));
         await dispatch(OrphanedRecordActions.fetchPage({
```
Comment (larskendall) [inline packages/client/src/components/pages/home-page/home-page.tsx:145]: Can this `fetchPage` call be removed? It looks like that may have been there as a means of getting `totalRowsCount` for the orphaned records count, which shouldn't be necessary now.

**Review by larskendall — CHANGES_REQUESTED** [review]

File: packages/client/src/components/pages/home-page/home-page.tsx (hunk 3 above)
Comment (richard3d) [reply packages/client/src/components/pages/home-page/home-page.tsx]: done

**Pull request merged**